README.md

```diff
@@ -5,23 +5,291 @@
+[profiles](https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/), which act as a starting point for
+an Istio install and can be customized by creating customization overlay files or passing parameters when
+calling helm. Similarly, the operator API uses the same profiles (expressed internally through the new API), which can be selected
+as a starting point for the installation. For comparison, the following example shows the command needed to install
+Istio using the SDS configuration profile using Helm:
+
+```bash
+helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+```
+
+In the new API, the same profile would be selected through a CustomResource (CR):
+
+```yaml
+# sds-install-cr.yaml
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: istio-operator-config
+  namespace: istio-system
+spec:
+  profile: sds
+```
+
+CRs are used when running the operator as a controller in a pod in the cluster. When using operator CLI mode and passing the 
+configuration as a file (see [Select a profile](#Select_a_profile)), only the spec portion is required.
+
+```yaml
+# sds-install.yaml
+
+profile: sds
+```
+
+If you don't specify a configuration profile, Istio is installed using the `default` configuration profile. All 
+profiles listed in istio.io are available compiled in, or `profile:` can point to a local file path to reference a custom
+profile base to use as a starting point for customization. See the [API reference](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto)
+for details.
+
+## Developer quick start
+
+The quick start describes how to install and use the operator `iop` CLI command. 
+
+#### Installation 
+
+```bash
+git clone https://istio.io/operator.git
+cd operator
+go build -o <your bin path> ./cmd/iop.go
+```
+
+#### Flags
+
+The `iop` command supports the following flags:
+
++   `logtostderr`: log to console (by default logs go to ./iop.go).
++   `dry-run`: console output only, nothing applied to cluster or written to files (default is true for now).
++   `verbose`: display entire manifest contents and other debug info (default is false).
+
+#### Basic default manifest
+
+The following command generates a manifest with the compiled in default profile and charts:
+
+```bash
+iop manifest
+```
+
+You can see these sources for the compiled in profiles in the repo under data/profiles, while the compiled in helm
+charts are under data/charts.
+
+#### Output to dirs
+
+The output of the manifest is concatenated into a single file. To generate a directory hierarchy with subdirectory
+levels representing a child dependency, use the following command:
+
+```bash
+iop manifest -o istio_manifests
+```
+
+Use depth first search to traverse the created directory hierarchy when applying your YAML files. Child manifest
```

Comment:
this block could be simplified...

Reply:
```suggestion
When you apply your YAML files, use depth first search to traverse the created directory hierarchy. This is needed because child manifest
```

Reply:
The suggestion has a slightly different meaning - I've updated the sentence to address what I think you're trying to address here.